modin/pandas/test/test_io.py

```diff
@@ -1837,27 +1838,29 @@ def test_read_sql(self, make_sql_connection):
         with warns_that_defaulting_to_pandas():
             pd.read_sql_table(table, conn)
 
-        # Test SQLAlchemy engine
         sqlalchemy_engine = sa.create_engine(conn)
-        eval_io(
-            fn_name="read_sql",
-            # read_sql kwargs
-            sql=query,
-            con=sqlalchemy_engine,
-        )
-
-        # Test SQLAlchemy Connection
         sqlalchemy_connection = sqlalchemy_engine.connect()
-        eval_io(
-            fn_name="read_sql",
-            # read_sql kwargs
-            sql=query,
-            con=sqlalchemy_connection,
-        )
+        if ReadSqlEngine.get() == "Connectorx":
```

Comment:
Please add a parameter that lists `ReadSqlEngine` values to this test and do `ReadSqlEngine.put(engine)` before running `pd.read_sql`.